### PR TITLE
[Doc Feature][Conversational bots in DOD][3438624]

### PR DESCRIPTION
**Pull request by v-npaladugu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/24aacbdce14d35da56526bcc7ec339f5f8e9ba3e/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 89 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/77f9aaab-8fb7-473a-a735-b2e17418f326) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/24aacbdce14d35da56526bcc7ec339f5f8e9ba3e/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/d9824de3-e67d-4309-bf5e-8c5a5985f100) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/24aacbdce14d35da56526bcc7ec339f5f8e9ba3e/msteams-platform%2Fwhats-new.md) | 94 | 44 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9edb2b5b-f6a5-4b8b-b0c3-e13f206331d9) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by v-habibaumme — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0a8f4103a915e5fb49ba82c6f96082007eff4b87/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 89 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/141c777b-fb22-49d7-8ae1-15856f1746d7) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/0a8f4103a915e5fb49ba82c6f96082007eff4b87/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3663e160-b568-4b29-ad2b-7651adc75521) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/0a8f4103a915e5fb49ba82c6f96082007eff4b87/msteams-platform%2Fwhats-new.md) | 94 | 44 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/42b29536-b6b4-4863-b25c-079c986b8258) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by rm3f — APPROVED** [review]

**Review by Wajeed-msft — DISMISSED** [review]

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Meetings enable collaboration, partnership, informed communication, and shared f
 <br>
 You can create scenes for meetings, provide notifications to users, populate in-meeting dialogs, and more with meeting app extensibility.
 
-Third-party and line-of-business apps built for meetings and calls are available only in Government Community Cloud (GCC) and GCC-High environments but aren't available for Department of Defense (DOD) environment.
+Third-party and line-of-business apps built for meetings and calls are available in Government Community Cloud (GCC), GCC-High, and Department of Defense (DOD) environments.
```
Comment (juna-gupta) [inline msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md:73]: We dont support third party apps in GCCH or DOD

**Review by juna-gupta — CHANGES_REQUESTED** [review]
Please address tjhe comment.

File: msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md (hunk 1 above)
Comment (v-npaladugu) [reply msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md]: Updated

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2c2a0d198c26df17cc7f08c751910cbdf6db48aa/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2702359e-93de-4126-9925-487247780660) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/2c2a0d198c26df17cc7f08c751910cbdf6db48aa/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e391fef7-1dda-4298-9bc3-4fd00c826781) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/2c2a0d198c26df17cc7f08c751910cbdf6db48aa/msteams-platform%2Fwhats-new.md) | 94 | 53 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c60a9557-65ef-4828-beb5-c0217ae7b27c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by rm3f — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/448139d5f94a954ebbb5ad3d12ced6b81f57fa7b/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80abffe2-b393-4b22-90c7-1713683c8a67) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/448139d5f94a954ebbb5ad3d12ced6b81f57fa7b/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b34aef0c-9639-4a81-a274-9a17debbba09) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/448139d5f94a954ebbb5ad3d12ced6b81f57fa7b/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7ba00b68-7ab5-4ee4-a7e0-e4e88795228c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/538ad94faf3865eb93c233dc856fcd4b400e4565/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2f04cd5f-1a4b-4b08-9659-a37d67c3fd7f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/538ad94faf3865eb93c233dc856fcd4b400e4565/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/56a715eb-e8d5-4638-bbd6-624b8881ec05) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/538ad94faf3865eb93c233dc856fcd4b400e4565/msteams-platform%2Fwhats-new.md) | 94 | 54 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e5e90cb8-ee49-45b2-9937-2d6c2e2d62aa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by rm3f — DISMISSED** [review]

**Review by juna-gupta — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a6683292476d6cb5c76b6b13f72a537cd86a3e6/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8292a498-06a3-4c90-9f35-4e6c3f0deca3) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/4a6683292476d6cb5c76b6b13f72a537cd86a3e6/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5317692a-71c7-4429-bc6d-de650ddeada8) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/4a6683292476d6cb5c76b6b13f72a537cd86a3e6/msteams-platform%2Fwhats-new.md) | 94 | 53 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4f660cf2-f994-4d35-a598-d415730fdff5) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/39b7389f19a72829bcecb36c1461b502ad3a5d90/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1277b9e8-8ccd-4282-b5ed-ab3619bbc51c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/39b7389f19a72829bcecb36c1461b502ad3a5d90/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6603a9f7-e1d1-48c4-ae5d-bbe785cd0118) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/39b7389f19a72829bcecb36c1461b502ad3a5d90/msteams-platform%2Fwhats-new.md) | 94 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/46350f1d-34cd-4f9e-9737-3dd3b029128b) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c7b8a219b69a8c7e912314db9d6317d837e418b/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/470ac087-dcf6-44db-b3e8-269fe0f8892f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/9c7b8a219b69a8c7e912314db9d6317d837e418b/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e0773c23-56ab-4dcf-bbc1-e3d04bef660e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9c7b8a219b69a8c7e912314db9d6317d837e418b/msteams-platform%2Fwhats-new.md) | 94 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/99dce651-63a9-482e-9cd0-5ab6b7a38372) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/apps-in-teams-meetings/teams-apps-in-meetings.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d63a6cdabf4c296b775e93a1e6ab732873abb84/msteams-platform%2Fapps-in-teams-meetings%2Fteams-apps-in-meetings.md) | 91 | 13 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/382c4d43-e637-4b9b-8aed-094dfaea3b32) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/mstdd-landing.yml](https://github.com/MicrosoftDocs/msteams-docs/blob/1d63a6cdabf4c296b775e93a1e6ab732873abb84/msteams-platform%2Fmstdd-landing.yml) | 92 | 3 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/12629575-336c-4d7b-86cf-352fee2bb141) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/1d63a6cdabf4c296b775e93a1e6ab732873abb84/msteams-platform%2Fwhats-new.md) | 94 | 55 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/84f0b067-498c-40fc-8efa-d16237ad1966) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Pull request merged**